docs/docs/enclave-configuration.md

```diff
@@ -316,184 +316,72 @@ The path should be absolute or relative to the root of the enclave module.
-            the [Windows subsystem for linux](https://docs.microsoft.com/en-us/windows/wsl/install), though this method
-            has not been extensively tested.
-
-    !!! tip
-        This process works best when the JDK used to generate the configuration files is the same version that will
-        subsequently be used to create the enclave's native image.
-
-    !!! info
-        For more information on running conclave targets on platforms other than linux, consult
-        [Enclave Modes](enclave-modes.md#system-requirements).
-
-6. Trigger `enclave` logic by running `client` requests:
-    ```
-    ./gradlew client:run
-    ```
-
-### Example using the executable JAR
-
-1.  To avoid the need to set up a linux environment using a VM or container, download
-    [GraalVM](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-21.2.0) for your operating system and
-    [install](https://www.graalvm.org/docs/getting-started/) it. Once you setup your GraalVM, ensure `native-image` is
-    installed:
+1. Create a `filter.json` file in the following path: `path/to/enclave/src/main/resources/META-INF/native-image/` 
+   with the following code to exclude unnecessary classes:
+   ```json
+   {
+   "rules": [
+   {"excludeClasses": "nonapi.**"},
+   {"excludeClasses": "com.r3.conclave.host.**"}
+   ]
+   } 
+   ```
+2. Download [GraalVM](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-21.2.0) for your operating system 
+   and [install](https://www.graalvm.org/docs/getting-started/) it.
 
+3. Install the `native image`:
     ```bash
     $JAVA_HOME/bin/gu install native-image
     ```
 
-    Also enable [native-image-agent](https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/#assisted-configuration-of-native-image-builds)
-    on the command line of the GraalVM `java` command:
-
-    ```bash
-    $JAVA_HOME/bin/java -agentlib:native-image-agent=config-output-dir=/path/to/enclave/src/main/resources/META-INF/native-image/ ...
-    ```
-
-1.  Add the [Shadow Gradle plugin](https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow)
-    to the `plugins` section of the `host`'s `build.gradle`:
+4. Add the [Shadow Gradle plugin](https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow) to the 
+   `plugins` section of the `host`'s `build.gradle` file:
 
     ```groovy
     plugins {
         id 'com.github.johnrengelman.shadow' version '6.1.0'
     }
     ```
 
-1.  Generate the shadow jar:
+5. Add the `EnclaveWebHost` main class to the `host`'s `build.gradle` file, *after* the plugins section:
 
+    ```groovy
+    project.mainClassName = "com.r3.conclave.host.web.EnclaveWebHost" 
+    ```
+   
+6. Generate the shadow jar:
     ```bash
     ./gradlew -PenclaveMode=mock host:shadowJar
     ```
+    This command creates an executable *shadow* JAR which contains all the host's and enclave's dependencies. You 
```

Comment:
I think we can write `host and enclave's`

Reply:
Rewrote.